pymc_marketing/mmm/mmm.py

```diff
@@ -615,6 +615,68 @@ def channel_contributions_forward_pass(
 
         return idata.posterior_predictive.channel_contributions.to_numpy()
 
+    def time_varying_channel_contributions_forward_pass(
```

Comment:
Can we just use the channel_contributions_forward_pass method? What is the benefit of having two separate ones?

Reply:
Yeap, makes sense. I'll fix it